lib/ex_unit/test/ex_unit/doc_test_test.exs

```diff
@@ -307,6 +373,11 @@ defmodule ExUnit.DocTestTest do
   end
 
   test "doctest failures" do
+
```

Comment:
Maybe we should say "the second line number in the doctest failure output tests" since we test whole output not only stacktrace (it actually has another stacktrace field).